scripts/deploy-zkevm-on-sequencer

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# This script is WIP
+#
+# 1. run: docker compose up --force-recreate --renew-anon-volumes
+#    in the espresso-sequencer repo.
+# 2. Run this script.
+# 3. run scripts/demo-test-end-to-end to test if it works.
+set -euo pipefail
```

Comment:
@sveitser is this still the entrypoint? I'm getting 
```
error: unexpected argument '--deploy' found

  tip: a similar argument exists: '--deploy-rollup-2'
```

Reply:
The script is out of date. There's no more `--deploy` flag.